src/redux/books/books.js

```diff
@@ -0,0 +1,50 @@
+const ADD = 'bookStore/books/ADD';
+const REMOVE = 'bookStore/books/REMOVE';
+const initialState = [
+  {
+    category: 'Action',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    progress: 64,
+    id: '1',
+  },
+  {
```

Comment:
Please ensure that you set the initial state with an empty array according to the project requirement.